### PR TITLE
Added skip validation and skip verify args to custom packs installer

### PR DESCRIPTION
**Pull request by ChanochShayner**
## Status
- [x] Ready

## Related Issues
fixes: https://github.com/demisto/etc/issues/45562

## Description
Added skip validation and skip verify args to custom packs installer.

## Screenshots
Paste here any images that will help the reviewer

## Does it break backward compatibility?
- [x] No

### CODE REVIEW

File: Packs/ContentManagement/Scripts/CustomPackInstaller/CustomPackInstaller.py [hunk 1]
```diff
@@ -6,11 +6,26 @@
...
+def build_url_parameters(skip_verify: bool, skip_validation: bool) -> str:
+    is_server_ge_to_6_5 = is_demisto_version_ge('6.5.0')
+    is_server_ge_to_6_6 = is_demisto_version_ge('6.6.0')
+
+    uri = '/contentpacks/installed/upload'
+    if skip_verify == 'true' and is_server_ge_to_6_5:
+        uri = f'{uri}?skipVerify=true'
+
+    if skip_validation == 'true' and is_server_ge_to_6_6:
+        uri = f'{uri}&skipValidation=true' if '?' in uri else f'{uri}?skipValidation=true'
```
Comment (guykeller) [inline Packs/ContentManagement/Scripts/CustomPackInstaller/CustomPackInstaller.py:18]: please refactor as we discussed to make this more readable

**Review by guykeller — CHANGES_REQUESTED** [review]

File: Packs/ContentManagement/Scripts/CustomPackInstaller/CustomPackInstaller.py (hunk 1 above)
Comment (ChanochShayner) [reply Packs/ContentManagement/Scripts/CustomPackInstaller/CustomPackInstaller.py]: done

**Review by guykeller — APPROVED** [review]

File: Packs/ContentManagement/ReleaseNotes/1_0_13.md [hunk 2]
```diff
@@ -0,0 +1,6 @@
+
+#### Scripts
+##### CustomPackInstaller
+- Added the **skip_verify** argument to skip pack signature validation, Available from 6.5.0 server version.
```
Comment (guykeller) [inline Packs/ContentManagement/ReleaseNotes/1_0_13.md:4]:
```suggestion
- Added the **skip_verify** argument to skip pack signature validation. Available from server version 6.5.0 .
```

File: Packs/ContentManagement/ReleaseNotes/1_0_13.md [hunk 3]
```diff
@@ -0,0 +1,6 @@
...
+- Added the **skip_validation** argument to skip all pack validations, use this flag just when migrate from custom content entities to custom content packs, Available from 6.6.0 server version.
```
Comment (guykeller) [inline Packs/ContentManagement/ReleaseNotes/1_0_13.md:5]:
```suggestion
- Added the **skip_validation** argument to skip all pack validations. This argument should be used only when migrating from custom content entities to custom content packs. Available from server version 6.6.0.
```

**Review by guykeller — CHANGES_REQUESTED** [review]

**Review by guykeller — APPROVED** [review]

**Pull request merged**